File: uHunt/src/easy/RelationalOperators.java
package easy;

//ID: 11172
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class RelationalOperators
{
	private static BufferedReader br;
	private static boolean test=true;
	
	public static void main(String[] args) throws IOException 
	{
		if(test)br=new BufferedReader(new FileReader(new File("./data/easy/RelationalOperators.in")));
		else br=new BufferedReader(new InputStreamReader(System.in));
		solucionarProblema();
		br.close();
	}

	private static void solucionarProblema() throws IOException 
	{
		int casos=Integer.parseInt(br.readLine()), num1, num2;
		String[] param;
		char rta;
		
		while(casos--!=0)
		{
			param=br.readLine().split(" ");
			num1=Integer.parseInt(param[0]);
			num2=Integer.parseInt(param[1]);
			rta=(num1<num2)?'<':'>';
			rta=(num1==num2)?'=':rta;
			System.out.println(rta);
		}
	}
	
}

File: uHunt/C++src/easy/RelationalOperators.cpp
using namespace std;
#include <cstdio>

bool test=true;

int main()
{
    if(test) freopen("../../data/easy/RelationalOperators.in","r",stdin);

    int cases, num1, num2;

    scanf("%d",&cases);
    while(cases--)
    {
        scanf("%d %d",&num1,&num2);
        char rta=(num1<num2)?'<':'>';
        rta=(num1==num2)?'=':rta;

        printf("%c\n",rta);
    }
    return 0;
}
